systemd/knxd.conf

```diff
@@ -11,8 +11,10 @@ KNXD_OPTS="-e 0.0.1 -E 0.0.2:8 -u /tmp/eib -b ip:"
 #  multicast client (-b ip:).
 # knxd's own bus address is 0.0.1; it will assign 0.0.2…0.0.9 to clients.
 # The knxd.socket file also tells knxd to listen to
-#  /run/eib (socket activation via systemd)
+#  /run/knx (socket activation via systemd)
 #  TCP port 6720 (socket activation via systemd)
+# The knxnetip.socket file also tells knxd to listen for KNXnet/ip connection on
+#  /run/knxnetip (socket activation via systemd)
```

Comment:
I called it that way because `KNXnet/IP` is what it's called in ISO 22510 and the protocol is exactly the TCP protocol described there, but if you prefer `knxnet`, I can change that.

Reply:
Please do.

Reply:
I've changed the name.